src/js/pages/Login/LoginPage.js

```diff
@@ -20,6 +20,8 @@ class Welcome extends React.Component {
       this.props.updateLanguage(storedLanguage);
     }
 
+    Object.values(Languages).map((lng, ind) => { debugger;})
```

Comment:
did you just leave that debugger there by mistake?

Reply:
yes the whole line should not be there, good catch!!